app/src/main/java/ch/epfl/sdp/peakar/camera/CameraActivity.java

```diff
@@ -173,6 +185,8 @@ public void takePictureListener(View view) throws IOException {
         Bitmap bitmap = CameraUtilities.combineBitmaps(cameraBitmap, compassBitmap);
         //Store the bitmap on the user device
         StorageHandler.storeBitmap(this, bitmap);
+        //Set visibility to invisible again after 100ms
```

Comment:
```suggestion
        //Set visibility to invisible again after FLASH_TIME_MS
```
I think this comment does not match the actual sleep time and could be confusing in future if we expect a 100ms time delay but it actually waits just 5ms. Maybe we should just leave the constant so one can go up and see the actual value.